BodoSQL/bodosql/tests/test_types/test_s3_tables_iceberg_catalog.py

```diff
@@ -105,7 +105,7 @@ def read_results(con_str, schema, table_name):
                     S3TablesCatalog(
                         "s3_tables_catalog",
                         **{"s3tables.warehouse": s3_tables_catalog.warehouse},
-                    ).drop_table(f"write_namespace.{table_name}")
+                    ).purge_table(f"write_namespace.{table_name}")
```

Comment:
Is `drop_table` renamed to `purge_table` generally?

Reply:
No it was just recommended in the the PR for S3TablesCatalog